lib/routemap.c

```diff
@@ -677,7 +677,10 @@ static unsigned int route_map_dep_hash_make_key(void *p);
 static void route_map_clear_all_references(char *rmap_name);
 static void route_map_rule_delete(struct route_map_rule_list *,
 				  struct route_map_rule *);
-static int rmap_debug = 0;
+bool rmap_debug;
+
+#define RMAP_DEBUG true
```

Comment:
just remove this and #define IS_RMAP_DEBUG (rmap_debug)

Reply:
Additionally if we are going to have a IS_RMAP_DEBUG define we either need to use it consistently or not.  Pick one for your tests.